extensions/amp-bind/0.1/bind-expression.js

```diff
@@ -51,7 +51,7 @@ const FUNCTION_WHITELIST = (function() {
      * @param {...?} items
      */
     /*eslint "no-unused-vars": 0*/
-    'copyAndSplice': function(array, start, deleteCount, items) {
+    'copyAndSplice': function copyAndSplice(array, start, deleteCount, items) {
```

Comment:
Perhaps we should just remove the wrapping `BindArray` object? Otherwise a typo between the key name and the function name is a possible source of errors.